resources/assets/components/Tags/index.js

```diff
@@ -52,15 +52,22 @@ class Tags extends React.Component {
       'hide-in-gallery': 'Hide In Gallery 👻',
       'good-for-sponsor': 'Good For Sponsor',
       'good-for-storytelling': 'Good For Storytelling',
+      'irrelevant': 'Irrelevant',
+      'inappropriate': 'Inappropriate',
+      'unrealistic-quantity': 'Unrealistic Quantity',
+      'test': 'Test',
+      'incomplete-action': 'Incomplete Action',
     };
 
+    const h2ClassName = this.props.disabled ? 'disabled' : 'enabled';
+
     return (
       <div>
-        <h4>Tags</h4>
+        <h4 className={h2ClassName}>Tags</h4>
```

Comment:
It is a little confusing to name this `h2ClassName` when it is styling an `h4` tag. Maybe we could call it like `showTags` which would describe the `enabled` or `disabled` status.

Reply:
love this idea!